test/images/issue-74839/.gitignore

```diff
@@ -0,0 +1 @@
+/issue-74839
```

Comment:
Should we maybe give this folder a more meaningful name? e.g. invalid-packet-test

Reply:
I was thinking calling it regression-issue-74839 to indicate it's a regression test.

Reply:
Sure, that sounds better.

Reply:
we can add more details in the README.